spec/serializers/user_serializer_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserSerializer do
+  describe '#avatar_url' do
+    it 'returns the avatar url based on the email' do
+      user = Fabricate(:user, email: 'hoang.mirs@gmail.com')
+      serializer = described_class.new(user)
+
+      expected_url = 'https://api.adorable.io/avatar/hoang.mirs@gmail.com'
```

Comment:
Not sure about having a real network request for this test as it can break any time if the service is down 💣  Either use VCR or a mock for this

Reply:
I didn't make any requests here, the avatar URL is a string and I just check if it returns the correct URL

Reply:
Oh right 🤦 No issues there indeed